tools/dotcms-cli/cli/src/main/java/com/dotcms/api/client/files/PushService.java

```diff
@@ -33,21 +31,17 @@ List<TraverseResult> traverseLocalFolders(
             boolean ignoreEmptyFolders, final boolean failFast);
 
     /**
-     * Processes the tree nodes by pushing their contents to the remote server. It initiates the push operation
-     * asynchronously, displays a progress bar, and waits for the completion of the push process.
+     * Processes the tree nodes by pushing their contents to the remote server. It initiates the
+     * push operation asynchronously, displays a progress bar, and waits for the completion of the
+     * push process.
      *
-     * @param output             the output option mixin
-     * @param workspace          the workspace path
-     * @param localPathStructure the local path structure of the folder being pushed
-     * @param treeNode           the tree node representing the folder and its contents with all the push information
-     *                           for each file and folder
-     * @param treeNodePushInfo   the push information summary associated with the tree node
-     * @param failFast           true to fail fast, false to continue on error
-     * @param maxRetryAttempts   the maximum number of retry attempts in case of error
+     * @param output           the output option mixin
+     * @param maxRetryAttempts the maximum number of retry attempts in case of error
+     * @param treeNodePushInfo
```

Comment:
can you please add the doc for the `treeNodePushInfo` and `params`? Part of the existing doc was removed (`treeNodePushInfo`). I would even rename the `params` param